### DISCUSSION TITLE
so-import-pcap: zeek - display urlencoded-form field

### DISCUSSION THREAD

**Discussion by rezpc** [Security-Onion-Solutions/securityonion#9493] [Unsupported Versions]
Grabbed this [pcap](https://github.com/Security-Onion-Solutions/securityonion/files/10326260/victim.zip) from Kringlecon22 involving a brute force password login on a website, thought it'd be fun to import it into SO 2,3 to hunt for the answers. In Kibana I see it imported correctly (via so-import-pcap), pretty easy to navigate the zeek logs to find most of the answers, however I was not able to detect the user/password attempts being logged in zeek:

![image](https://user-images.githubusercontent.com/82719188/210112085-8b285ce1-62fc-45dd-bd18-7933ac8dad89.png)

The field value in wireshark is urlencoded-form:

![image](https://user-images.githubusercontent.com/82719188/210111999-aeee5f02-7688-441b-8771-c3e250029908.png)

Is this just operator error and I'm looking in the wrong part of Kibana or is this information not being captured/indexed?

**Comment by dougburks** [comment]
Zeek doesn't log those details by default, but you could write a Zeek script to configure it to do so. 

Here's another option without having to resort to Zeek scripts or Wireshark. If you use the URL provided by so-import-pcap, then it will take you our SOC interface where you can view not only the Zeek logs as you did in Kibana but also pivot to the full TCP stream. For example:
![image](https://user-images.githubusercontent.com/1659467/210359142-53ac3e77-9204-4ad6-bf61-d084257e97af.png)
